.gitignore

```diff
@@ -34,3 +34,4 @@ vendor/
 .idea.bak/
 .vscode/
 tests/
+/goworld.ini
```

Comment:
why remove this file?

Reply:
This file is constantly edited across different projects. However, it's content can be reproduced directly by copying from the `goworld.ini.sample`. Removing it from the source file prevents others from accidentally committing non-relevant change, or worse, breaking others' project.
I've adding process to the build script, specifically handling creation of the `goworld.ini` from sample, and placing it to the `bin` folder. So most people won't even notice it.